src/formats/script.c

```diff
@@ -238,19 +234,16 @@ static bool parse_tag(sd_script_tag *new, str *src, int *now) {
     }
 
     // Check if tag is legit.
-    str test;
     for(int m = 3; m > 0; m--) {
-        str_from_slice(&test, src, *now, *now + m);
-        if(str_equal_c(&test, "usw") && find_numeric_span(src, *now + m) > *now + m) {
+        // str_from_slice(&test, src, *now, *now + m);
+        if(strncmp(src->data + *now, "usw", m) == 0 && find_numeric_span(src, *now + m) > *now + m) {
```

Comment:
```suggestion
        if(strncmp(str_c(src) + *now, "usw", 3) == 0 && find_numeric_span(src, *now + m) > *now + m) {
```
The strncmp call should compare all 3 characters like the str_equal_c did before, not just the first m.